src/gcodeExport.cpp

```diff
@@ -246,10 +249,6 @@ void GCodeExport::setFlavor(EGCodeFlavor flavor)
     {
         firmware_retract = true;
     }
-    else 
-    {
-        firmware_retract = false;
-    }
```

Comment:
I'd say the best solution would be to eliminate the REPRAP volumetric machine type and then have machine_firmware_retract and machine_volumetric in the machine definitions.  The patch to Cura would still be useful for dynamic determination of volumetric mode.